### ISSUE TITLE
Crash from missing texture

### ISSUE THREAD

**Issue by lilleyse** [KhronosGroup/COLLADA2GLTF#181]
[a318.dae.txt](https://github.com/KhronosGroup/COLLADA2GLTF/files/1936025/a318.dae.txt)

This model seems to have texture references but does not link or embed any texture files. This is causing a crash [here](https://github.com/KhronosGroup/COLLADA2GLTF/blob/master/GLTF/src/GLTFTexture.cpp#L34) due to `source` being a null pointer.

The dae was exported from this blender file: https://github.com/kalmykov/fr24-3d-models/blob/master/source/a318/a318.blend

Brought up on the Cesium forum: https://groups.google.com/forum/#!topic/cesium-dev/Oa7qbuL4G2E

**Comment by lasalvavida** [comment]
That file is not valid COLLADA. Frankly, I'm surprised OpenCOLLADA accepts it at all.

```bash
xmllint --noout --schema http://www.khronos.org/files/collada_schema_1_4_1.xsd a318.dae

a318.dae:13: element library_images: Schemas validity error :

Element '{http://www.collada.org/2005/11/COLLADASchema}library_images': Missing child element(s). 

Expected is one of ( {http://www.collada.org/2005/11/COLLADASchema}asset, {http://www.collada.org/2005/11/COLLADASchema}image ).
```

Among other errors related to invalid element names.

I can add a NULL check and maybe a warning somewhere, since we don't want to crash.

**Comment by lasalvavida** [comment]
Fixed in #183.

![image](https://user-images.githubusercontent.com/2489570/39100797-947df18c-465d-11e8-87e0-a60cf9a373cf.png)
